### PR TITLE
LRDOCS-3856 Creating Theme Color Schemes tutorial

### PR DESCRIPTION
**Pull request by mwilliams2014**
cc @blzaugg

### CODE REVIEW

File: develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    Specifying a default color scheme lets you return to the default look and 
+    feel for your theme.
+    
+    Note that color schemes are sorted alphabetically by `name` rather than `id`. 
+    For example, a color scheme named `Day` and `id` `02` would be selected by 
+    default over the `Default` color scheme with `id` `01`. The `<default-cs>` 
+    element overrides the alphabetical sorting and sets the color scheme that is 
+    selected by default, when the theme is chosen. Adding this element to the 
+    default color scheme ensures that it is selected when the theme is chosen.
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: > For example, a color scheme named `Day` and `id` `02` would be selected by default over the `Default` color scheme with `id` `01`.

This is confusing/contradictory.

File: develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    
+    The `<color-scheme-images-path>` element specifies where theme thumbnail 
+    images are located (you only have to place this element in one color scheme 
+    for it to affect all of them). For example you could use the folders
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: > (you only have to place this element in one color scheme for it to affect all of them)

Do we need to mention this?

I assume this isn't by design. What happens if they point to different paths?

Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: That's how it was documented in earlier versions, so I assume that it is the design. I would imagine you could point to different paths if you wanted to of course. I figure by taking this approach though, you only have to define that element once, and then forget about it. Also, this approach keeps everything organized under one folder.

File: develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown (hunk 1 above)
Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: Yeah, it's unfortunate wording. The default color scheme (the original color scheme) is named 'default.' I don't believe you can change that. the original color scheme isn't selected by default though, if you have another color scheme that comes before it alphabetically, so you have to set that element to true to ensure that it is. Sounds like a bug.

Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: Actually, it looks like the original color scheme doesn't have to be named `Default.` I'll update it.

Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: Can we reword that than? To make the distinction clear.

Maybe add a Note the explains the weirdness?

File: develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown (hunk 2 above)
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: How about "...(place this element in the first color scheme for it to affect all of them)..."?

File: develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown (hunk 1 above)
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: Cool

Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: Take that back, looks like the original is named default, but you can provide a separate color scheme if you wish and make that the default instead, essentially placing your default color scheme styles in their own CSS file.

Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown]: Only the `<css-class>` element has to be `default`. The name is arbitrary. I'll clarify that.

**Comment by mwilliams2014** [comment]
Updated @blzaugg let me know if anything else need changed. Thx!

**Comment by blzaugg** [comment]
LGTM

**Comment by sez11a** [comment]
View just my changes: https://github.com/sez11a/liferay-docs/compare/mwilliams2014:75a0ae3dae...42bf628e83
View total diff: https://github.com/sez11a/liferay-docs/compare/d9d8e8186b...42bf628e83

**Pull request closed without merging**